dio/lib/src/adapter.dart

```diff
@@ -73,30 +86,15 @@ class ResponseBody {
   int statusCode;
 
   /// Returns the reason phrase associated with the status code.
-  /// The reason phrase must be set before the body is written
-  /// to. Setting the reason phrase after writing to the body.
+  /// The reason phrase must be set before the body is written to.
+  /// Setting the reason phrase after writing to the body.
```

Comment:
This sentence still does not make any sense to me.

Reply:
Updated with value explanations.